src/components/App.js.jsx

```diff
@@ -3,8 +3,23 @@ import { Link } from 'react-router'
 import '../css/App.css'
 import BuilderContainer from './containers/BuilderContainer.js.jsx'
 import DashboardContainer from './containers/DashboardContainer.js.jsx'
+import Ajax from './helpers/ajax.js'
+import { connect } from 'react-redux' 
 
 class App extends Component {
+
+  componentDidMount() {
+    const api = new Ajax()
+    api.fetchTags().then(response => this.storeTags(response.data))
```

Comment:
@wfro wouldn't the above example return a function reference that's expecting 2 more arguments? Don't we want to just pass the return of `fetchTags` to our dispatch action?

Everything I've tried just keeps returning a pending promise to our component. I've tried binding the response to a function and returning that function reference to call in my component, I've tried creating a local variable to save the AJAX response to and returning that. I must not be understanding how promises handle callbacks correctly or something.